src/runtime/dom-extras.ts

```diff
@@ -19,6 +20,7 @@ export const patchPseudoShadowDom = (
   patchSlotInsertAdjacentText(hostElementPrototype);
   patchTextContent(hostElementPrototype, descriptorPrototype);
   patchChildSlotNodes(hostElementPrototype, descriptorPrototype);
+  patchSlotRemoveChild(hostElementPrototype);
```

Comment:
Just wanna call out that this will _only_ be patched for **`scoped`** components. The associated issues had some examples using no encapsulation, but we're moving away from patching this behavior for that case.